tests/test_batch.py

```diff
@@ -68,6 +76,46 @@ def test_create_batch_and_fetch_results(self, request_mock):
             assert job.result == self.job_result
         assert request_mock.last_request.method == "GET"
 
+    def test_get_batch(self, request_mock):
+        batch_requested = self.sdk.get_batch(self.batch.id)
+        assert (
+            batch_requested.id == self.batch.id
+        )
+
+    def test_cancel_batch_endpoint(self, request_mock):
+        self.batch.cancel()
+        assert request_mock.last_request.method == "PUT"
+        assert (
+            request_mock.last_request.url
+            == f"{self.sdk._client.endpoints.core}/api/v1/batches/{self.batch.id}/cancel"
+        )
+
+    def test_cancel_batch_success(self, request_mock):
+        self.batch.cancel()
+        assert (
+            self.batch.status == 'CANCELED'
+        )
+
+    def test_add_job_and_get_job(self, request_mock):
```

Comment:
We already have a test for the add_job method. This test can focus on the get_job part I think!